File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-03/plagiarized/L6/06/cabang.kt
package IR

import java.util.Scanner

fun main() {
    val input = Scanner(System.`in`)
    var w: Double
    var f: Double
    var i: Double
    var h: Double
    var TotalBmi: Double

    print("Enter weight in pounds : ")
    w = input.nextDouble() // inputan weight

    print("Enter feet : ")
    f = input.nextDouble() // inputan feet

    print("Enter inches : ")
    i = input.nextDouble() // inputan inches

    h = f * 12 + i // menghitung tinggi
    TotalBmi = hitBMI(w, h) // menghitung BMI

    // menampilkan hasil
    println("BMI is $TotalBmi")
    result(TotalBmi)
}

fun hitBMI(w: Double, h: Double): Double {
    return (w * 0.45359237) / ((h * 0.0254) * (h * 0.0254))
}

fun result(TotalBmi: Double) {
    when {
        TotalBmi > 30 -> println("Obese ")
        TotalBmi < 25 && TotalBmi > 18.5 -> println("Normal ")
        TotalBmi > 25 && TotalBmi < 30 -> println("Overweight ")
        else -> println("Underweight ")
    }
}

File: datasets/IR-Plag/convertai-original-java-rest-python/case-03/plagiarized/L6/06/cabang.py
import math

def main():
    w = float(input("Enter weight in pounds: "))
    f = float(input("Enter feet: "))
    i = float(input("Enter inches: "))
    h = f * 12 + i  # calculate height
    total_bmi = hit_bmi(w, h)  # calculate BMI
    # display results
    print("BMI is", total_bmi)
    result(total_bmi)

def hit_bmi(w, h):
    return (w * 0.45359237) / ((h * 0.0254) * (h * 0.0254))

def result(total_bmi):
    if total_bmi > 30:
        print("Obese")
    elif total_bmi < 25 and total_bmi > 18.5:
        print("Normal")
    elif total_bmi > 25 and total_bmi < 30:
        print("Overweight")
    else:
        print("Underweight")

if __name__ == "__main__":
    main()

